src/workerd/api/sockets.h

```diff
@@ -62,8 +62,8 @@ class PipelinedAsyncIoStream final : public kj::AsyncIoStream, public kj::Refcou
 };
 
 struct SocketOptions {
-  bool tsl; // TODO(later): TCP socket options need to be implemented.
-  JSG_STRUCT(tsl);
+  jsg::Unimplemented tls; // TODO(later): TCP socket options need to be implemented.
```

Comment:
Sorry it took me so long to get to this, but I did actually make the same change in https://github.com/cloudflare/workerd/pull/237/files#diff-60b97f1a91fb6495e0c3c870b0f479e52fd448c80b9c445a06795d006f369ca8R14